wagtail/project_template/home/templates/home/welcome_page.html

```diff
@@ -0,0 +1,377 @@
+{% templatetag openblock %} load i18n {% templatetag closeblock %}
+<link rel="stylesheet" type="text/css" href="/static/admin/css/fonts.css">
+<style type="text/css">
+  body, main {
+    margin: 0 auto;
+  }
+  .body, .tip {
```

Comment:
As far as I can see neither of those two classes are present in the HTML.

Reply:
Removed in 959d41b.